ui/src/app/analytics/counters/counters.component.html

```diff
@@ -1,35 +1,48 @@
-<form #refreshRateForm="ngForm" class="form-inline" name="refreshRateForm" role="form" (ngSubmit)="changeRefreshRate()" novalidate>
+<form #refreshRateForm="ngForm" class="form-inline" name="refreshRateForm" role="form" (ngSubmit)="changeRefreshRate()"
+      novalidate>
   <div class="form-group">
     <label for="refreshRateFormField" style="display: inline">Update [seconds]:</label>
-    <input #refreshRateFormField2="ngModel" type="number" id="refreshRateFormField" name="refreshRateFormField" autofocus [(ngModel)]="refreshRateFormField" required
-           class="form-control" placeholder="Inerval in seconds" style="width: 150px"/>
+    <input #refreshRateFormField2="ngModel" type="number" id="refreshRateFormField" name="refreshRateFormField"
+           autofocus [(ngModel)]="refreshRateFormField" required
+           class="form-control input-sm" placeholder="Inerval in seconds" style="width: 100px"/>
     <span class="help-block" *ngIf="refreshRateFormField2.invalid && refreshRateFormField2.touched">Please provide a valid polling interval (0-1000s).</span>
   </div>
-  <button type="submit" [disabled]="refreshRateForm.invalid"
-          class="btn btn-default">Update <span class="glyphicon glyphicon-arrow-right"></span></button>
```

Comment:
The glyphicon didn't survive the refactoring. Is it intentional? If not, "arrow right" is probably not the best image... think there is one for refresh that might be more suitable.